backend/app/services/quiz.py

```diff
@@ -0,0 +1,12 @@
+from typing import Sequence
+
+from sqlalchemy import select
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.models.quiz import Quiz
+
+
+async def get_quizzes(session: AsyncSession) -> Sequence[Quiz]:
```

Comment:
I though about creating a repository or DAO, but for this size of project it would be an overkill

Reply:
Yeah i agree 100%